packages/shared/src/components/cards/v1/CollectionCard.tsx

```diff
@@ -81,12 +78,10 @@ export const CollectionCard = forwardRef(function CollectionCard(
             >
               {title}
             </CardTitle>
-            {tagsOnCard && (
-              <>
-                <div className="flex flex-1" />
-                <PostTags tags={post.tags} />
-              </>
-            )}
+            <>
+              <div className="flex flex-1" />
+              <PostTags tags={post.tags} />
+            </>
```

Comment:
This should also work? We needed the `<></>` because it was inside `()`
```suggestion
            <div className="flex flex-1" />
            <PostTags tags={post.tags} />
```

Reply:
Yep!